composeApp/src/commonMain/kotlin/org/ooni/probe/domain/GetTestDescriptors.kt

```diff
@@ -19,13 +19,13 @@ import org.ooni.probe.data.models.toDescriptor
 
 class GetTestDescriptors(
     private val getDefaultTestDescriptors: () -> List<DefaultTestDescriptor>,
-    private val listInstalledTestDescriptors: () -> Flow<List<InstalledTestDescriptorModel>>,
+    private val listInstalledTestDescriptors: (Boolean?) -> Flow<List<InstalledTestDescriptorModel>>,
     private val descriptorUpdates: () -> Flow<DescriptorUpdatesStatus>,
     private val getPreferenceValues: (List<SettingsKey>) -> Flow<Map<SettingsKey, Any?>>,
 ) {
-    operator fun invoke(): Flow<List<Descriptor>> {
+    operator fun invoke(isExpired: Boolean? = null): Flow<List<Descriptor>> {
```

Comment:
I don't think there's a case we'll want to get just the expired test descriptors with all the extra data this is fetching. So I think we could avoid having this argument here, so we don't need to set it as false in a lot of places.

Reply:
I think we need the argument. include expired tests or not to include expired tests